Formula/n/neo4j-mcp.rb

```diff
@@ -0,0 +1,30 @@
+class Neo4jMcp < Formula
+  desc "Neo4j official Model Context Protocol server for AI tools"
+  homepage "https://neo4j.com/docs/mcp/current/"
+  url "https://github.com/neo4j/mcp/archive/refs/tags/v1.1.0.tar.gz"
+  sha256 "ae622058f73f862162b2a0c51f33b3f50ae972844af3218c62ae7136e864b5d2"
+  license "GPL-3.0-or-later"
+  head "https://github.com/neo4j/mcp.git", branch: "main"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = "-s -w -X main.Version=v#{version}"
+    system "go", "build", *std_go_args(ldflags: ldflags), "./cmd/neo4j-mcp"
+  end
+
+  test do
+    # Test version output
+    assert_match version.to_s, shell_output("#{bin}/neo4j-mcp --version")
+
+    # Test help output shows expected options
+    help_output = shell_output("#{bin}/neo4j-mcp --help")
+    assert_match "Neo4j Model Context Protocol Server", help_output
+    assert_match "NEO4J_URI", help_output
```

Comment:
can we test the capacities like other mcp server?

Reply:
Implemented capabilities checks in the latest push.